Update installation and import instructions for ObjC and Swift (#20)
cossacklabs/product-docs: themis/languages/objc/features.md

> In order to use ObjCThemis, you need to import its headers:

Reviewer: ```suggestion
In order to use ObjCThemis, you need to import it:
```
Since there are no more headers to import or include.
